src/js/pyodide.ts

```diff
@@ -395,6 +401,9 @@ export async function loadPyodide(
   if (Module.exited) {
     throw Module.exited.toThrow;
   }
+  if (options.pyproxyToStringRepr) {
+    API.setPyProxyToStringMethod("repr");
```

Comment:
Why are we entering a string value rather than a boolean value here, and converting it inside the function?

Reply:
Meant to say `true` here.

Reply:
I'm surprised typescript is okay with this...